_literate/A-ensembles/tutorial.jl

```diff
@@ -38,7 +38,7 @@ knn = machine(knn_model, X, y)
 # and fit it
 
 fit!(knn, rows=train)
-ŷ = predict(knn, X[test, :]) # or use rows=test
+ŷ = MLJ.predict(knn, X[test, :]) # or use rows=test
```

Comment:
Generally, if there is a name conflict with an MLJ method, I just `import` the offending package. (We do this a lot with DataFrames because it exports `transform`.) I haven't run the code, but I'm stumped as to what package is giving you a conflict. Which is it?

Reply:
I don't remember that the specific package was mentioned it just threw the error that it couldn't find `predict`. Now quite surprisingly, I have tried running again without `MLJ.` and it worked so maybe we can take this off.